street-explorer/www/js/lane_editor.js

```diff
@@ -0,0 +1,205 @@
+import { downloadGeneratedFile } from "./files.js";
```

Comment:
I adapted this from street explorer and am sharing some common code. Does that seem OK and maintainable?